src/main/java/io/xpring/xpring/XpringException.java

```diff
@@ -0,0 +1,43 @@
+package io.xpring.xpring;
+
+/**
+ * Represents exceptions thrown by Xpring components of the Xpring SDK.
+ */
+public class XpringException extends Exception {
+  /**
+   * Input entities given to a Xpring component were attached to different networks.
+   * <p>
```

Comment:
I've been seeing html tags appear in my own javadocs... I'm wondering if these are intentional or automatic, and either way, if they are necessary?